packages/kbn-capture-oas-snapshot-cli/src/kibana_worker.ts

```diff
@@ -52,25 +56,25 @@ export type Result = 'ready';
     watch: false,
   };
 
+  let root: Root;
+
+  set(settings, 'migrations.skip', true);
+  set(settings, 'elasticsearch.skipStartupConnectionCheck', true);
+
   if (serverless) {
     // Satisfy spaces config for serverless:
     set(settings, 'xpack.spaces.allowFeatureVisibility', false);
     set(settings, 'xpack.spaces.allowSolutionVisibility', false);
-    const { startKibana } = createTestServerlessInstances({
-      kibana: { settings, cliArgs },
-    });
-    await startKibana();
+    root = createServerlessKibana(settings, cliArgs);
   } else {
-    const root = createRootWithCorePlugins(settings, cliArgs);
-    await root.preboot();
-    await root.setup();
-    await root.start();
+    root = createRootWithCorePlugins(settings, cliArgs);
   }
-
-  const result: Result = 'ready';
-
-  process.send(result);
+  await root.preboot();
+  await root.setup();
+  const { http } = await root.start();
+  const oas = await http.generateOas({ baseUrl: 'http://localhost:5622', filters });
```

Comment:
This is the change where most savings come from. Because we already have `Root.setup` which gives us the internal HTTP start contract we can directly invoke `generateOas` rather than waiting for Kibana to start and connect to ES.

Kibana cannot respond to any traffic bc security/licensing needs to ES to inject the license signature.